### ISSUE TITLE
Broken wiki links

### ISSUE THREAD

**Issue by brianjmurrell** [storaged-project/blivet#979]
Two of the three links at https://github.com/storaged-project/blivet/wiki are broken.

Where can documentation on the end-user capabilities of blivet[-gui] be found?  Can it resize partitions and/or LVM LVs and everything that sits on top of them right up to and including the filesystem -- without data loss for example?

**Comment by vojtechtrefny** [comment]
I updated the broken link, but we currently have only developer documentation for both [blivet](http://storaged.org/blivet/) and [blivet-gui](http://storaged.org/blivet-gui/) so not really useful for users (I have some old and incomplete [user documentation](https://github.com/storaged-project/blivet-gui/tree/master/help/en-US/blivet-gui) for blivet-gui, but that's not available online).

We unfortunately don't have list of features or capabilities for blivet. But yes, we do support resizing of both logical volumes and partitions.

**Comment by brianjmurrell** [comment]
It seems that it's not as functional as it could/should be with **live** devices.  I cannot seem to resize an ext4 filesystem on an LV for example.  I can do that using CLI tools though.

Is my understanding above correct?  If so, is more _live_ functionality on the roadmap?  Is there a roadmap that can be viewed anywhere?

**Comment by vojtechtrefny** [comment]
> It seems that it's not as functional as it could/should be with **live** devices. I cannot seem to resize an ext4 filesystem on an LV for example. I can do that using CLI tools though.
> 
> Is my understanding above correct?

Yes, we unfortunately don't support resize for mounted devices even if the filesystem supports online resize. A lot of missing functionality in blivet is result of it starting as storage library for the Anaconda installer -- everything is unmounted during the installation.

> If so, is more _live_ functionality on the roadmap? Is there a roadmap that can be viewed anywhere?

We track some feature requests/enhancements here in the [issues](https://github.com/storaged-project/blivet/issues?q=is%3Aissue+is%3Aopen+label%3Aenhancement) and we also have a [TODO](https://github.com/storaged-project/blivet/blob/3.4-devel/TODO) list, but there is a "real" roadmap for blivet.

**Comment by vojtechtrefny** [comment]
> If so, is more _live_ functionality on the roadmap?

I've realized I didn't actually answer this part of the question. Yes, we plan to add more options for "configuring" existing storage and support more filesystems in the future.
